native/src/components/__tests__/EmbeddedOffers.spec.tsx

```diff
@@ -0,0 +1,72 @@
+import { DateTime } from 'luxon'
+import React from 'react'
+
+import { MALTE_HELP_FORM_OFFER_ALIAS, SPRUNGBRETT_OFFER_ALIAS } from 'shared'
+import { CategoryModel, OfferModel } from 'shared/api'
+
+import render from '../../testing/render'
+import EmbeddedOffers from '../EmbeddedOffers'
+
+jest.mock('../SprungbrettOffer.tsx', () => ({
+  __esModule: true,
```

Comment:
:thinking: Is the `.tsx` and the `__esModule: true` really needed? I have never seen/needed this.

Reply:
Yes, we need it because we're mocking a default export. There's a tiny bit more information about it here: https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options

Reply:
Hmmm, okay. Now I am confused why we don't have to use this in a lot of other places...

Reply:
Oh maybe its because you are returning an object with default instead of just returning the mock directly?

Reply:
I'm not sure how often we mock the default export of one of our own files 🤷 Looks like we so far only use it in the `DashboardNavigationTiles`.

Reply:
Ooooh, you're right, I found a better way, thanks!